cmd/observe/agent_events.go

```diff
@@ -91,18 +91,18 @@ func getAgentEventsRequest() (*observerpb.GetAgentEventsRequest, error) {
 		if err := since.CheckValid(); err != nil {
 			return nil, fmt.Errorf("failed to convert `since` timestamp to proto: %v", err)
 		}
-		// Set the until field if both --since and --until options are specified and --follow
-		// is not specified. If --since is specified but --until is not, the server sets the
-		// --until option to the current timestamp.
-		if selectorOpts.until != "" && !selectorOpts.follow {
-			ut, err := hubtime.FromString(selectorOpts.until)
-			if err != nil {
-				return nil, fmt.Errorf("failed to parse the until time: %v", err)
-			}
-			until = timestamppb.New(ut)
-			if err := until.CheckValid(); err != nil {
-				return nil, fmt.Errorf("failed to convert `until` timestamp to proto: %v", err)
-			}
+	}
+	// Set the until field if both --since and --until options are specified and --follow
+	// is not specified. If --since is specified but --until is not, the server sets the
+	// --until option to the current timestamp.
+	if selectorOpts.until != "" && !selectorOpts.follow {
```

Comment:
This comment should be update as with the change, `--until` may be provided without `--since`.

Reply:
Nice catch, thanks! Updated in [9909183](https://github.com/cilium/hubble/pull/792/commits/9909183b871af12a3c64669b6a78693d27aaf539), let me know if the wording is ok or if there is anything to improve :).